models/mongo/notes.go

```diff
@@ -0,0 +1,50 @@
+package mongo
+
+import (
+	"context"
+	"notes-service/models"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type note struct {
+	ID       string         `json:"id" bson:"_id,omitempty"`
+	AuthorId string         `json:"authorId" bson:"authorId,omitempty"`
+	Title    *string        `json:"title" bson:"title,omitempty"`
+	Blocks   []models.Block `json:"blocks" bson:"blocks,omitempty"`
+}
```

Comment:
What's the reasoning behind having this `note` structure within the `mongo` package? Can't it be defined within the `models` package and then used by `mongo`?